src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx

```diff
@@ -60,68 +69,100 @@ const PatientView = (props: {
     return (
         <>
             <Row gutter={[1, 40]} justify="center">
-                <Col span={12}>
+                <Col span={8}>
                     <Card
                         key={props.patient.id}
                         className="patient-card"
                         title={<Title level={4}>{name}</Title>}
                         type="inner"
-                        hoverable
                         bordered
                     >
                         <div className="info-container">
                             <div className="info-left">
-                                <h4>Personnummer:</h4>
-                                <span>{props.patient.identifier[0].value}</span>
-                                <h4>Kjønn:</h4>
-                                <span>
-                                    {props.patient.gender !== 'male' &&
-                                    props.patient.gender !== 'female'
-                                        ? props.patient.gender
-                                              .charAt(0)
-                                              .toUpperCase() +
-                                          props.patient.gender.slice(1)
-                                        : props.patient.gender === 'male'
-                                        ? 'Mann'
-                                        : 'Kvinne'}
-                                </span>
-                                <h4>Alder:</h4>
-                                <span>
-                                    {props.patient.birthDate !== undefined ? (
-                                        calcAge()
-                                    ) : (
-                                        <div>Ikke oppgitt</div>
-                                    )}
-                                </span>
-                            </div>
+                                <div className="item-container">
+                                    <h4>
+                                        <IdcardOutlined className="field-icon" />
+                                        Personnummer:
+                                    </h4>
 
+                                    <p>{props.patient.identifier[0].value}</p>
+                                </div>
+                                <div className="item-container">
+                                    <h4>
+                                        <ManOutlined className="field-icon" />
```

Comment:
Should this not only be the man's gender icon but depend on gender or be gender-neutral?

Reply:
I was hoping to find an icon for multigender, but ant design doesnt have it. I could either make it conditional on m/f/x, or i could find an icon that suits all posibilities from another source. What do you think? Personally i think the latter would fit more.

Reply:
For example one of these: https://www.iconfont.cn/search/index?searchType=icon&q=gender

Reply:
Agree, If you don't find an icon in exact same style I have previously gotten help from Idean to design the icon!

Reply:
For now, i just made a small conditional for male/female. The API doesnt seem to accept other values to the gender field, so this is sufficient for now. Will expand later if needed.